src/ASM/FiniteElement.h

```diff
@@ -114,6 +114,8 @@ class MxFiniteElement : public FiniteElement
 protected:
   //! \brief Writes the finite element object to the given output stream.
   virtual std::ostream& write(std::ostream& os) const;
+  //! \brief Global Output stream operator.
+  friend std::ostream& operator<<(std::ostream& os, const MxFiniteElement& fe);
```

Comment:
yes, i need the implementation. it prints as a FiniteElement otherwise. just try for yourself.